package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fixtable-ember",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Could you merge MyPureCloud/fixtable-ember#master into your fork (or rebase) so that diffs like this go away? Otherwise, if the version gets bumped before this PR merges, your PR will reset the version to 1.1.1.